graylog2-server/src/main/java/org/graylog2/shared/journal/LocalKafkaJournal.java

```diff
@@ -795,6 +796,14 @@ public long getNextReadOffset() {
         return nextReadOffset;
     }
 
+    public double getJournalUtilization() {
```

Comment:
Could you add some JavaDoc explaining what the returned value means? Also, it would be really helpful to know the unit of the returned value, e.g., that it's a percentage and not a ratio with a value between `0` and `1`.